api/swagger.yaml

```
@@ -2882,6 +2882,11 @@ definitions:
             type: "object"
             additionalProperties:
               type: "string"
+          # Applicable to UNIX platforms
+          Capabilities:
+            type: "array"
+            description: |
+              A list of kernel capabilities to be available for container (this overrides the default set).
```

Can you add `items` and `items -> type` here? Perhaps add an example value while we're at it;

```suggestion
              A list of kernel capabilities to be available for container (this overrides the default set).
            items:
              type: "string"
            example:
              - "CAP_NET_RAW"
              - "CAP_SYS_ADMIN"
              - "CAP_SYS_CHROOT"
              - "CAP_SYSLOG"
```